### DISCUSSION TITLE
[rejected] W API sale/offers brakuje info o produktach.

### DISCUSSION THREAD

**Discussion by leszekdubiel** [allegro/allegro-api#4920] [Suggestions - rejected]
Allegro oczekuje, że 70% moich ofert będzie podłączone z produktami, więc potrzebuję raport wszystkich ofert, gdzie każda oferta ma zaznaczone czy jest powiązana z produktem czy nie jest. 

```
-- oferta 1  -- ma produkt? -- tak, produkt id .... 
-- oferta 2 -- ma produkt -- nie, null 
-- oferta 3 -- ma produkt? -- nie, null 
-- oferta 4 -- .... 
-- ... 
... 
-- oferta n -- ma produkt?  -- tak, produkt id ... 
```

Ponieważ ```sale/offers``` nie zawiera informacji o produktach, to muszę pytać dwa razy: 

-- najpierw o wszystkie oferty 
-- a potem o wszystkie oferty z filtrem ```/sale/offers?product.id.empty=true```

Ba! Muszę pytać wiele, wiele razy, bo przecież jest stronicowanie... 

Dlatego API ```sale/offers``` pownno zwracać chociaż informację, czy produkt jest podłączony. Aby nie spamować allegro zapytaniami i móc łatwiej programować.

**Comment by Aryak2099** [comment]
No a nie możesz filtrować żeby pobierało tylko te które nie mają productid?
